tests/test_gpu_common.py

```diff
@@ -1343,6 +1350,25 @@ def test_npthreads(self):
             assert op.arguments(time_M=2, npthreads0=5)
 
 
+class TestMisc(object):
+
+    def test_pickling(self):
```

Comment:
That should be in `test_pickle` but I guess it's easier for gpu to dso it here

Reply:
mmhh in my view, the GPU tests are "more specific" than the pickling tests

Just like some MPI tests are found outside of `test_mpi.py`